src/components/Disputes/DisputeCard.js

```diff
@@ -15,18 +17,12 @@ function DisputeCard({ dispute, onSelectDispute }) {
 
   return (
     <CardItem onClick={handleClick}>
-      <div
-        css={`
-          display: flex;
-          align-items: center;
-          justify-content: space-between;
-        `}
-      >
+      <div>
         <DisputeStatus dispute={dispute} />
```

Comment:
Bit of a nitpick, but does this need to be enclosed in this div if we don't apply any styling to it?

Reply:
Actually the `<DisputeCard />` is styling it to be `100% width`. If we remove the div, it would be applied to the dispute status tag